yucca/umbrel-app.yml

```diff
@@ -0,0 +1,29 @@
+manifestVersion: 1.1
+id: yucca
+category: automation
+name: Yucca
+version: "0.10.0"
+tagline: Simple NVR solution video surveillance
+description: >-
+  Simple NVR solution video surveillance for home and business.  
+  Perfect for both a solution for safe cities with thousands of cameras and launching a subscriber service, as well as for home use self-hosted.
+
+releaseNotes: >-
+  Init Yucca on Umbrel
+
```

Comment:
For this initial release we can just leave the releaseNotes key as a placeholder:
`releaseNotes: ""`